lib/collection/ui/GameViewTemplate.tsx

```diff
@@ -135,6 +136,9 @@ export const GameViewTemplate = (props: {
                 <DropdownMenu.Item key="end" onSelect={props.onEnd}>
                   <DropdownMenu.ItemTitle>End match</DropdownMenu.ItemTitle>
```

Comment:
Maybe change this to "Finish match" since both options technically do end it, just differently? Not a huge deal